tests/test_logstash_notifier.py

```diff
@@ -205,6 +205,48 @@ def test_get_user_data(self):
                 message['user_data']
             )
 
+class SupervisorOutPutLoggingTestCase(BaseSupervisorTestCase):
+    """
+    Test capturing stdout/stderr logs.
+    """
+
+    def test_output_logging(self):
+        """
+        Test output logging.
```

Comment:
This docstring adds no value. How about something like "Test stdout is captured in logs when capture-output argument is set"